menas/ui/service/EntityDialog.js

```diff
@@ -253,9 +260,12 @@ class MappingTableDialog extends EntityDialog {
       this.oController.byId("newMappingTableName"));
     let hasValidSchema = EntityValidationService.hasValidSchema(oMT, "Mapping Table",
       this.oController.byId("schemaVersionSelect"));
-    let hasValidHDFSPath = this.oController.byId("addMtHDFSBrowser").validate();
+    let hasValidHDFSPath = EntityValidationService.hasValidHDFSPath(oMT.hdfsPath,
+      "Mapping Table HDFS path",
+      this.oController.byId("selectedHDFSPathLabel"));
+    let hasExistingRawHDFSPath = hasValidHDFSPath ? this.oController.byId("addMtHDFSBrowser").validate() : false;
```

Comment:
```suggestion
    let hasExistingHDFSPath = hasValidHDFSPath ? this.oController.byId("addMtHDFSBrowser").validate() : false;
```
`Raw` here is misleading for `MappingTables`